resources/did-configuration/index.html

```diff
@@ -647,8 +649,10 @@ <h2>
         <ul>
           <li>The object MUST contain an `id` property, and its value MUST be a valid DID URL reference.</li>
           <li>The object MUST contain a `type` property, and its value MUST be the string `"LinkedDomains"`.</li>
-          <li>The object MUST contain an `origins` property, and its value MUST be an array of one or more 
-            <a href="https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy">domain origins</a>.</li>
+          <li>The object MUST contain a `serviceEndpoint` property, and its value MUST be an object that 
+              contains a `origin` property, the value of which MUST be an array of one or more 
```

Comment:
I know this feels like it could be more generic, but IMO this is actually better that its custom... @csuwildcat can you add an example of the entire service defintion to this PR?

Reply:
Yeah +1 to an example